packages/eslint-config/package.json

```diff
@@ -14,5 +14,8 @@
     "node": "16.x || 18.x",
     "npm": "7.x || 8.x"
   },
-  "main": "lib"
+  "main": "lib",
+  "peerDependencies": {
+    "eslint": "^8.27.0"
```

Comment:
The reason I chose `8.27.0` rather than the latest version is that this is the minimum version that we need installed. This is because [this rule](https://eslint.org/docs/latest/rules/no-new-native-nonconstructor) was only added in `8.27.0` so any earlier version of `eslint` will error

Reply:
Fair point 😌